### PR TITLE
refact/#108: content card

### LINKED ISSUE #108: refact content card
- use of Material UI components
- add option with picture

### PR DESCRIPTION
**Pull request by kindOfCurly**
- refactored design of content card
- fixed some warnings
- added a light grey background

closes #108

### CODE REVIEW

File: src/components/pageComponents/ContentCard.js [hunk 1]
```diff
@@ -18,14 +21,11 @@ import { Link } from "react-router-dom";
...
   card: {
-    border: "solid 2px #00817B",
     marginBottom: "16px",
   },
 
   cardContent: {
-    "&:last-child": {
-      paddingBottom: "16px",
-    },
+    paddingTop: "0px",
```
Comment (Naarij) [inline src/components/pageComponents/ContentCard.js:28]: If there is a 0px padding on top, then the attribute is redundant I think

File: src/components/pageComponents/InfoMessageCard.js [hunk 2]
```diff
@@ -20,7 +16,6 @@ const useStyles = makeStyles((theme) => ({
     color: "#2E303C",
     textAlign: "start",
     margin: "0",
```
Comment (Naarij) [inline src/components/pageComponents/InfoMessageCard.js:18]: As above (i know this isnt from you)

**Review by Naarij — APPROVED** [review]
Great work! Looks pretty good now.
Just my two cents: I think the Card Action Area should (if possible) be the complete card except the like button for usability. I was first confused when i clicked on the header of the card and the content didn't showed up.

File: src/components/pageComponents/ContentCard.js (hunk 1 above)
Comment (kindOfCurly) [reply src/components/pageComponents/ContentCard.js:28]: This acutally overrides the default 16px-padding (which looked very strange).

File: src/components/pageComponents/InfoMessageCard.js (hunk 2 above)
Comment (kindOfCurly) [reply src/components/pageComponents/InfoMessageCard.js:18]: Good point. Thank you!

**Comment by kindOfCurly** [comment]
> 
> 
> Great work! Looks pretty good now.
> Just my two cents: I think the Card Action Area should (if possible) be the complete card except the like button for usability. I was first confused when i clicked on the header of the card and the content didn't showed up.

I think this would be inconsistent with using the CardHeader Component: it is not clickable but has one action (our favorite button)... imo I wouldn't change this behaviour.

**Pull request merged**